### PR TITLE
Run RPMInspect in tomcatjss CI

### PR DESCRIPTION
**Pull request by ckelleyRH**

### CODE REVIEW

File: tests/bin/rpminspect.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# Don't run metadata check as we can't know the build host subdomain
+# of CI runners in advance to add to an allow list
+
+echo "Running RPMInspect on SRPM"
+rpminspect-fedora -E metadata /github/home/build/tomcatjss/SRPMS/*.rpm
```
Comment (edewata) [inline tests/bin/rpminspect.sh]: The absolute path here kind of limits the usability of this script. I think it should be possible to change the build command in the test like this:
```
./build.sh --work-dir=build rpm
```
then you can use a relative path like this:
```
rpminspect-fedora -E metadata build/SRPMS/*.rpm
```

**Review by edewata — APPROVED** [review]
Just one suggestion, feel free to update/merge.

**Review by fmarco76 — APPROVED** [review]
I agree with @edewata on the relative path. Besides the path looks good.

File: tests/bin/rpminspect.sh (hunk 1 above)
Comment (ckelleyRH) [reply tests/bin/rpminspect.sh]: Sure, will update, thanks!

**Comment by ckelleyRH** [comment]
@edewata @fmarco76 Thanks!

**Pull request merged**